### ISSUE TITLE
cwd on terminal

### ISSUE THREAD

**Issue by rosshinkley** [chjj/blessed#176]
The terminal defaults to - and is seemingly not changeable from - `$HOME`.  Is there a reason why a `cwd` option is not included?  I feel like I'm missing something obvious.

As an example, I'm thinking of a use case close to the multiplexer example, but instead of having both terminals open to `$HOME`, having one or both open to an optional `cwd` option that defaults to `$HOME`.

**Comment by ngotchac** [comment]
I agree, but it is still achievable with : `bottomleft.pty.write('cd /tmp\n', 'utf8');`
